src/editors/containers/ErrorAlerts/ErrorAlert.jsx

```diff
@@ -5,7 +5,7 @@ import { Alert } from '@edx/paragon';
 import { Info } from '@edx/paragon/icons';
```

Comment:
This move is straining the name `container` for the parent directory. I agree it needs to be a shared component, not inside text editor, but interested in what to call it.